src/codec.ts

```diff
@@ -171,18 +179,18 @@ export function bytesToString (buf: Uint8Array): string {
 /**
  * String -> Uint8Array
  */
```

Comment:
The doc comment differs from the implementation now.

Reply:
updated comment accordingly